### PR TITLE
react1-week1/nada

### PR DESCRIPTION
**Pull request by NadaJrad**

### CODE REVIEW

File: 7_react/react1/week1/todo-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+import './App.css';
+ 
+export default function TodoListApp (){
```
Comment (BJORN-VINTHER) [inline 7_react/react1/week1/todo-app/src/App.js]: Consider to give the method the same name as the component or file. As developer I will expect `App.js` to export a function named `App()`

File: 7_react/react1/week1/todo-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <li>{todo.description} {todo.deadline} </li>
+  );
+  return <ul>{listItems}</ul>;
+}
+
+const todoList = ([
+{ description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+{ description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+{ description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+]);
```
Comment (BJORN-VINTHER) [inline 7_react/react1/week1/todo-app/src/App.js:26]: Very clean and organized code 👍

**Review by BJORN-VINTHER — COMMENTED** [review]
All code looks good Nada. Good job 👍 

Try to make sure that your PR only contains changes for the homework you are handing in. In this PR there are changes for the database course. It can be a good idea to inspect the Files changed in the PR to see if you accidently committed something that you didn't mean to. If so, you can create a new commit where you remove those files.

File: 7_react/react1/week1/todo-app/src/App.js (hunk 2 above)
Comment (NadaJrad) [reply 7_react/react1/week1/todo-app/src/App.js:26]: thanks for your review i am really appreciate 😀.

**Pull request closed without merging**